.gitignore

```diff
@@ -5,7 +5,7 @@
 .tox
 .pytest_cache
 .eggs
-
+.vscode/*
```

Comment:
this should go in your global git ignore so you don't have to repeat it in every project